api/tests/routes/pro/patch_collective_offer_test.py

```diff
@@ -184,6 +188,30 @@ def test_patch_collective_offer_update_legacy_template(self, client):
         assert template.domains == [domain]
         assert template.interventionArea == ["01", "2A"]
 
+    def when_activating_all_existing_offers_work(self, app):
+        # Given
+        offer1 = CollectiveOfferFactory(isActive=False)
+        venue = offer1.venue
+        offerer = venue.managingOfferer
+        user_offerer = offerers_factories.UserOffererFactory(user__email="pro@example.com", offerer=offerer)
+
+        # When
+        client = TestClient(app.test_client()).with_session_auth("pro@example.com")
+        data = {
+            "isActive": True,
+            "offererId": user_offerer.offerer.id,
```

Comment:
(détail) On a déjà accès à `offerer`. Pas besoin d'utiliser la variable `user_offerer`. C'est plus concis.

```suggestion
            "offererId": offerer.id,
```

Idem dans d'autres fonctions de test que tu as ajoutées.